doc/devel/tag_guidelines.rst

```diff
@@ -20,56 +35,38 @@ Every gallery example should be tagged with:
 * 1+ content tags
 * structural, domain, or internal tag(s) if helpful
 
-Tags can repeat existing forms of organization (e.g. an example is in the Animation folder and also gets an ``animation`` tag).
+Tags can repeat existing forms of organization (e.g. an example is in the Animation
+folder and also gets an ``animation`` tag).
+
+Tags are helpful to denote particularly good "byproduct" examples. E.g. the explicit
+purpose of a gallery example might be to demonstrate a colormap, but it's also a good
+demonstration of a legend. Tag ``legend`` to indicate that, rather than changing the
+title or the scope of the example.
 
-Tags are helpful to denote particularly good "byproduct" examples. E.g. the explicit purpose of a gallery example might be to demonstrate a colormap, but it's also a good demonstration of a legend. Tag ``legend`` to indicate that, rather than changing the title or the scope of the example.
+.. card::
 
-**Tag Categories** - See :doc:`Tag Glossary <tag_glossary>` for a complete list of tags.
+    **Tag Categories**
+    ^^^
+    .. rst-class:: section-toc
 
-I. API tags: what content from the API reference is in the example?
-II. Structural tags: what format is the example? What context can we provide?
-III. Domain tags: what discipline(s) might seek this example consistently?
-IV. Internal tags: what information is helpful for maintainers or contributors?
+        .. toctree::
+            :maxdepth: 2
+
+            tag_glossary
```

Comment:
Note: Indentation is not needed ([from](https://docutils.sourceforge.io/docs/ref/rst/directives.html#toc-entry-40)):

> The "class" directive sets the classes attribute value on its content or on the first immediately following [...] element

It may be more readable to write this as
```
.. rst-class:: section-toc
.. toctree::
    :maxdepth: 2

    tag_glossary
```